### DISCUSSION TITLE
exe kapanıyor

### DISCUSSION THREAD

**Discussion by EsatAkkasoglu** [KebabLord/turkanime-indirici#37] [Genel] [answered]
Selamlar, exe dosyasından indirmeye çalışıyorum. Sürekli kapanıyor. 
![WhatsApp Image 2022-12-09 at 4 05 44 PM](https://user-images.githubusercontent.com/101052449/206710921-cd9e98ba-c888-4b08-98c7-2a602cb8c785.jpeg)

**Comment by KebabLord** [comment, marked as answer]
artık "çalışmıyor" olarak işaretlenmiş sürümlerden birini kullanıyorsun, yeni düzeltilmiş sürümü indirilebilirsin: https://github.com/KebabLord/turkanime-indirici/releases/tag/v7.0.4

bu arada lütfen sorunları issue bölümünde açalım
